### ISSUE TITLE
Accessibility: Dates form

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#593]
1. Grouped form controls missing an accessible name: Maybe the group could be called 'Date type' and then each of the buttons is labeled (Single date, Date range)-- there's also a typo in the actual toggle itself.
2. Please change the text at the top of the page to an H2: "Enter dates related to your deposit"

**Comment by justinlittman** [comment]
@astridu What is the typo?

**Comment by astridu** [comment]
Toggle says "Singe" instead of "Single"

**Comment by amyehodge** [comment]
@justinlittman it says "Singe" instead of "Single"

**Comment by justinlittman** [comment]
Ouch. Singed.
